web/src/nodes/paragraph.ts

```diff
@@ -18,18 +21,93 @@ import '../ui/nodes/properties/provenance/provenance'
 @customElement('stencila-paragraph')
 @withTwind()
 export class Paragraph extends Entity {
+  /**
+   * a list of parent nodes that can require different
+   * behaviour/rendering of this node.
+   */
+  static subscribedParentNodes = [
+    'ListItem',
+    'TableCell',
+    'QuoteBlock',
+    'Table',
+    'Figure',
+    'CodeChunk',
+  ]
+
   /**
    * In static view just render the `content`.
    */
   override renderStaticView() {
     return html`<slot name="content"></slot>`
   }
 
+  /**
+   * The ancester node directly above this one in the tree
+   */
+  private directAncestor: NodeType
+
+  /**
+   * A consumer controller for the `EnityContext`,
+   * used to subscribe to the parent node's `EnityContext` if needed.
+   */
+  private parentContext: ContextConsumer<
+    { __context__: EntityContext },
+    this
+  > | null = null
+
+  override connectedCallback() {
+    super.connectedCallback()
+
+    this.directAncestor = this.ancestors.split('.').reverse()[0] as NodeType
+
+    /* 
+      if this Paragraph needs to be subscribed to parent node
+      creates a consumer for the `entityContext`, 
+      this will subscribe to the nearest entityContext above this node.
+    */
+    if (
+      Paragraph.subscribedParentNodes.includes(this.directAncestor) &&
+      !this.parentContext
+    ) {
+      this.parentContext = new ContextConsumer(this, {
+        context: entityContext,
+        subscribe: true,
+      })
+      this.context = {
+        ...this.context,
+        cardOpen: this.parentContext.value?.cardOpen,
+      }
+    }
+  }
+
+  protected override update(
+    changedProperties: PropertyValueMap<this> | Map<PropertyKey, unknown>
+  ): void {
+    super.update(changedProperties)
+
+    if (this.parentContext) {
+      /* 
+        if `parentContext` is initiated,
+        mirror the paragraph entity's context `cardOpen` status to the parent.
+      */
+      if (this.parentContext.value?.cardOpen !== this.context.cardOpen) {
+        this.context = {
+          ...this.context,
+          cardOpen: this.parentContext.value.cardOpen,
+        }
+      }
+    }
+  }
+
   /**
    * In dynamic view render `content`, and `authors` and summary stats in a node card
    * that is shown on hover.
    */
   override renderDynamicView() {
+    if (Paragraph.subscribedParentNodes.includes(this.directAncestor)) {
+      return this.renderStaticView()
```

Comment:
In the relatively near future I'm going to suggest that we get rid of `renderStatic` etc and just have a single `render` method (as is normal). So, in anticipation of that, how about:

```suggestion
      return html`<slot name="content"></slot>`
```